tests/deployment/switch_config.py

```diff
@@ -53,17 +53,6 @@ def configure():
                                      'site_layout')
 
 
-@pytest.fixture
-def is_brocade():
-    """open the site-layout file to see if we have a brocade switch"""
-
-    with open('site-layout.json') as layout_data:
-        layout = json.load(layout_data)
```

Comment:
so the entire file is not using `json`, you could get rid of the `import json`

Reply:
right